gov2/sqs/DeleteQueue/README.md

```diff
@@ -0,0 +1,9 @@
+### DeleteQueue/DeleteQueuev2.go
+
+This example deletes an Amazon SQS queue.
+
+`go run DeleteQueuev2.go -q QUEUE-NAME`
+
+- _QUEUE-NAME_ is the name of the queue from which the message is deleted.
```

Comment:
...the name of the queue that is deleted.
(the example shows deleting a queue, not deleting a message)

Reply:
Good catch, fixed.